boards/arm/nrf51_blenano/Kconfig.board

```diff
@@ -6,4 +6,5 @@
 
 config BOARD_NRF51_BLENANO
 	bool "nRF51 BLENANO"
+	select HAS_DTS_GPIO
```

Comment:
Why here and not in the drivers/gpio Kconfig?

Reply:
I removed it and left it only in drivers/gpio/Kconfig.nrfx